lib/io/file.fz

```diff
@@ -0,0 +1,30 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of Fuzion standard library feature file
+#
+#
+# -----------------------------------------------------------------------
+
+# file -- effect related to fileio
```

Comment:
`io.file` should probably not be the effect, but just a unit type that contains effects related to files. The actual effects should be features like `io.file.read`, `io.file.write`, `io.file.delete`, `io.file.rename`, `io.file.file_size`, etc.